compose.sim.webots.yaml

```diff
@@ -0,0 +1,54 @@
+services:
+
+  rviz:
+    image: husarion/rviz2:humble-22-12-20
+    runtime: nvidia
+    container_name: rviz
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./config/rosbot.rviz:/root/.rviz2/default.rviz
+    environment:
+      - RMW_IMPLEMENTATION
+      - DISPLAY=${DISPLAY:?err}
+      - NVIDIA_VISIBLE_DEVICES=all
+      - NVIDIA_DRIVER_CAPABILITIES=all
+
+  rosbot-xl:
+    image: husarion/webots:galactic
+    runtime: nvidia
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+    environment:
+      - RMW_IMPLEMENTATION
+      - DISPLAY=${DISPLAY:?err}
+      - NVIDIA_VISIBLE_DEVICES=all
+      - NVIDIA_DRIVER_CAPABILITIES=all
+    command: ros2 launch webots_ros2_husarion robot_launch.py robot_name:="rosbot_xl"
+
+  mapping:
+    image: husarion/slam-toolbox:humble-22-12-07
+    network_mode: host
+    ipc: host
+    volumes:
+      - ./config/slam_toolbox_params_webots.yaml:/slam_params.yaml
+    environment:
+      - RMW_IMPLEMENTATION
+    command: >
+      ros2 launch slam_toolbox online_sync_launch.py
+        slam_params_file:=/slam_params.yaml
+        use_sim_time:=True
+
+  map-saver:
+    image: husarion/nav2-map-server:humble-22-12-07
+    network_mode: host
+    ipc: host
+    volumes:
+      - ./maps:/maps
+    environment:
+      - DDS_CONFIG
```

Comment:
Not sure if `DDS_CONFIG` is necessary, and if it is, it should be also included in other containers.

Reply:
Added in every service